### DISCUSSION TITLE
Node.js and npm command not found

### DISCUSSION THREAD

**Discussion by carlo2222** [community/community#53401] [Packages]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi! I installed nodejs and nmv for windows. When I run the ionic framework from the command panel it always says "npm not found". I've reinstalled both several times. I think there are problems in creating the link for the packages. how can i solve?
![2023-04-20_21-40-54-8687328c55c90b39a53b4ac81604c011](https://user-images.githubusercontent.com/129618933/233614770-57ae3981-5c94-4f8a-8661-5bde5be51370.jpg)
![Senza titolo-1](https://user-images.githubusercontent.com/129618933/233615098-0c4de5a3-9ac3-4551-a593-d4d26407ed48.jpg)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @carlo2222 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
